WORKSPACE

```diff
@@ -659,6 +659,15 @@ dist_http_archive(
 # grpc_extra_deps().
 load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
 
+# for patching the "com_github_cncf_udpa" deps loaded by grpc_deps
+dist_http_archive(
```

Comment:
You should move this before `load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")` to make sure our definitions can take precedence.